external/kafka/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_${scala.binary.version}</artifactId>
-      <version>0.8.0</version>
```

Comment:
That being said, @helena may soon update this version anyway in https://github.com/apache/spark/pull/3631 IIUC.